test/integration/users.test.js

```diff
@@ -1206,6 +1206,7 @@ describe("Users", function () {
           if (err) {
             return done(err);
           }
+          expect(fetchStub.calledOnce).to.be.equal(true);
```

Comment:
since adding role calls fetch and if no user is found there will only be 1 fetch call of fetching users from discord server there wont any other fetch call that will be made for updating roles